src/components/GoToLink.ts

```diff
@@ -0,0 +1,30 @@
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
+
+type Props = {
+  isbacklink?: string;
+};
+
+export const GoToLink = styled(NavLink)<Props>`
+  display: flex;
+  justify-content: center;
+  color: ${({ theme }) => theme.textSecondary};
+  text-decoration: none;
+  cursor: pointer;
+  font-size: 1.1rem;
+  font-weight: 500;
+  display: flex;
+  transition: all 0.3s ease-in-out;
+
+  span {
+    font-weight: 900;
+    margin-left: ${({ isbacklink }) => (isbacklink == 'true' ? '0' : '0.5rem')};
+    margin-right: ${({ isbacklink }) => (isbacklink == 'true' ? '0.5rem' : '0')};
```

Comment:
```suggestion
    margin-left: ${({ isbacklink }) => (isBackLink == 'true' ? '0' : '0.5rem')};
    margin-right: ${({ isbacklink }) => (isBackLink == 'true' ? '0.5rem' : '0')};
```

Nie wiem czemu pascal case nie użyłeś. Ponadto dziwny jest ten conditional styling. Jak już chciałeś 2 style naraz ograć, to polecam coś takiego:

```suggestion
    margin-left: 0.5rem;
    margin-right: 0;

   ${({isBackLink}) => isBackLink && css`
    margin-left: 0;
    margin-right: 0.5rem;
  `
```